script/testing/junit/src/FilterTrace.java

```diff
@@ -33,70 +31,80 @@ public static void main(String[] args) throws Throwable {
         Connection conn = db.getDbTest().newConn();
         List<String> tab = getAllExistingTableName(mog,conn);
         removeExistingTable(tab,conn);
+        List<String> aggregations = new ArrayList<>();
+        aggregations.add("COUNT");
+        aggregations.add("MAX");
+        aggregations.add("MIN");
+        aggregations.add("AVG");
+        aggregations.add("SUM");
         while (mog.next()) {
             String cur_sql = mog.sql.trim();
+            for(int i=0; i<mog.comments.size();i++){
+                if(mog.comments.get(i).contains(Constants.SKIPIF)||mog.comments.get(i).contains(Constants.ONLYIF)){
+                    skip_flag = true;
+                    break;
+                }
+            }
+            if(cur_sql.contains("SELECT")){
+                boolean contain_aggregation = false;
+                for(String i: aggregations){
+                    if(cur_sql.contains(i)){
+                        contain_aggregation = true;
+                    }
+                }
+                if(!contain_aggregation){
+                    skip_flag = true;
+                }
```

Comment:
What is this doing? Why are aggregations skipped?

Reply:
For this part we skip queries that don't contain aggregation keywords. I was using this to filter out traces from the sqlite files, but then realized that the sqlite syntax is a lot different